### PR TITLE
Module read_data

### PR DESCRIPTION
**Pull request by andpet0101**
This is the pull request for the module read_data:

- Created a new branch from current main
- Copied text from the original https://github.com/ktrns/scrnaseq
- Fixed bug in SaveSeuratRds when an assay has multiple matrix directories
- Changed plots to use patchwork
- Changed png device from bugged ragg to cairo

### CODE REVIEW

**Comment by andpet0101** [comment]
Could you please review and merge in the correct order? Thanks :-)

File: R/functions_io.R [hunk 1]
```diff
@@ -1487,40 +1481,148 @@ ParsePlateInformation = function(cell_names, pattern='_(\\d+)_([A-Z])(\\d+)$') {
   return(plate_information)
 }
 
+####################################################################################
+# This is a copy of the SeuratObject::SaveSeuratRds with the following bugs fixed: #
+# - matrices consisting of multiple paths (submatrices) cannot be copied           #
+####################################################################################
+SaveSeuratRds_Fixed <- function (object, file = NULL, move = TRUE, destdir = deprecated(), relative = FALSE, ...)
```
Comment (ktrns) [inline R/functions_io.R:1488]: Why is this called `_Fixed`? Are you planning to keep the old function?

File: _quarto.yml [hunk 2]
```diff
@@ -67,10 +67,13 @@ format:
     #  message: false
```
Comment (ktrns) [inline _quarto.yml:67]: In the end (not necessarily now) I'd like to clean up this file and remove options we don't use anyway. We won't ever convert this report into PDF, so why keep the options :)?

File: modules/1_read_data/read_data.qmd [hunk 3]
```diff
@@ -1030,69 +1011,120 @@ for (a in Seurat::Assays(sc)) {
...
-#| label: fig-read_data_basic_summary_{{assay}}_{{i}}
-#| fig-asp: 0.5
+#| label: fig-read_data_basic_summary_{{assay}}
 #| fig-cap: '{{caption}}'
+#| fig-height: 8
 #| echo: false
 #| warning: false
 
-print(plist[[{{i}}]])
+p
```
Comment (ktrns) [inline modules/1_read_data/read_data.qmd:1048]: :-D

**Review by ktrns — APPROVED** [review]
Thanks for the work! I just skimmed over the changes, and approve to move on to the next PR.

**Pull request merged**